### DISCUSSION TITLE
Suwayomi / Tachidesk

### DISCUSSION THREAD

**Discussion by MatthewGlenn** [selfhst/icons#63] [New Icon Request]
Github Repos:
https://github.com/Suwayomi/Suwayomi-Server
https://github.com/Suwayomi/docker-tachidesk

Icon:
![](https://github.com/Suwayomi/Suwayomi-Server/raw/master/server/src/main/resources/icon/faviconlogo.png)

**Comment by shollyethan** [comment]
Suwayomi has been added - thanks!

**Reply by MatthewGlenn** [reply]
No problem, thanks for maintaining this repo!

**Comment by LordFransie** [comment]
@shollyethan  The icon you have on the site is out of date. You're using the old logo and not the included logo.
